pg/scripts/init-schema.sql

```diff
@@ -0,0 +1,43 @@
+-- ma-exec pg psql -f scripts/init-schema.sql -U postgres
+
+create table enzymes (
+    protein varchar(16) not null,
+    organism varchar(4) not null,
+    domain varchar(7),
+    ko char(6),
+    reaction_id char(6),
+    ec_number varchar(255) not null,
```

Comment:
This should be checked with the researchers.
1 EC number has 8 characters, so with a 2 char separator that would make room for at most 25 EC numbers. Likely this is a valid cap, but it's worth double-checking.